File: iryw.py
import numpy as np
import time



def createAndSumArr(x: int):

    start = time.time()

    arr = np.empty(x, dtype = np.int64)


    for i in range(x):
        arr[i] = i
    
    return np.sum(arr), (time.time()-start)


sum, t = createAndSumArr(100000000)

print(f"Sum -> {sum}, Time -> {t}")

File: iryw.go
package main

import (
	"fmt"
	"time"
)

func createAndSumArr(x int) (int64, float64) {

	//start time
	start := time.Now()

	//arr
	arr := make([]int64, x)

	//fill arr
	for i := 0; i < x; i++ {
		arr[i] = int64(i)
	}

	//calculate arrSum

	var arrSum int64 = 0

	for _, num := range arr {
		arrSum += num

	}

	return arrSum, (time.Since(start).Seconds())

}

func main() {
	sum, t := createAndSumArr(100000000)

	fmt.Printf("Sum -> %d, Time -> %f", sum, t)
}
